src/js/panels.js

```diff
@@ -566,7 +569,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         },
         components: {
             attachTooltipOnLang: {
-                type: "gpii.firstDiscovery.panel.lang.attachTooltipOnLang",
+                type: "gpii.firstDiscovery.attachTooltip",
```

Comment:
Thanks for double checking and figuring out the special css class for language tooltips is no longer needed. 

I'm thinking, the use of the sub-component ```attachTooltipOnLang``` was to narrow down the context for the demands block so the special tooltip class could only be applied to language tooltips. Since this narrow-down is no longer needed, we should be able to add ```gpii.firstDiscovery.attachTooltip``` as a direct grade into the language panel component instead of making it a subcomponent.